spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/StreamDefinitionRepository.java

```diff
@@ -28,6 +29,7 @@
  * @author Mark Fisher
  */
 @Repository
+@NoRepositoryBean
```

Comment:
Yes, it would.  Since I put the new repository interface in the same package as the others, this annotation is to indicate to the Spring Data infrastructure not to create an implementation (as we create one 'by hand' now).